api/azure-pipelines.ts

```diff
@@ -1,42 +1,130 @@
-import cheerio from 'cheerio'
 import got from '../libs/got'
 import { createBadgenHandler, PathArgs } from '../libs/create-badgen-handler'
+import { isNullOrUndefined } from 'util'
 
 export default createBadgenHandler({
   title: 'Azure Piplines',
   examples: {
-    '/azure-pipelines/yarnpkg/yarn/Yarn Acceptance Tests': 'build',
-    '/azure-pipelines/yarnpkg/yarn/Yarn Acceptance Tests/azure-pipelines': 'build (branch)',
+    '/azure-pipelines/build/status/yarnpkg/yarn/Yarn Acceptance Tests': 'build',
+    '/azure-pipelines/build/status/yarnpkg/yarn/Yarn Acceptance Tests/azure-pipelines': 'build (branch)',
```

Comment:
It would be nice to have more examples showcasing all these features 👍

Reply:
Added build version example. I don't have any Azure DevOps public project which I can use as an example. Need to figure out how to put examples.

Reply:
@deepdotnet Can we use `yarnpkg/yarn` for all examples?

Reply:
Yes, I tried, I didn't find any release info in API response. If we can add example UTs, ATs, and release in yarnpkg/yarn, I can add examples here.

Reply:
At least we would need a working example to test if the handler works as expect.